### ISSUE TITLE
In Visual Studio 2017 15.5.3 .. some of the backend projects are shown as incompatible

### ISSUE THREAD

**Issue by IoTFier** [Microsoft/SmartHotel360-Azure-backend#12]
Hello, I pulled the backend repo this morning. After opening the solution in the Visual Studio 2017 15.5.3 on a PC, I see 7 services project shown as incompatible and doesn't open those projects.

Anyone experiencing this?

Screenshot as attached -
![image](https://user-images.githubusercontent.com/26844611/34884573-c03652c6-f782-11e7-95da-994e7fdbddcf.png)

Thanks

**Comment by eiximenis** [comment]
Not sure what is happening. Everything should load in VS 15.5...

**Comment by IoTFier** [comment]
I agree, but I didn't experience that. So is the request to find if anyone had similar experience.

**Comment by bradygmsft** [comment]
@IoTFier - have you been able to try with VS 15.5. We built this solution targeting features available in 15.5. Please try and let us know if you still have problems with that version. If so, we'll re-open this, but for now I'll close this as "by design" since we are targeting 15.5. Thanks!

**Comment by IoTFier** [comment]
Hi @bradygmsft , I have been using 15.5. I opened this issue with 15.5.3, and in the last week updated my VS to 15.5.4. I haven't seen the above issue ever since I reloaded projects within IDE option under solution explorer. Thanks
